cds_migrator_kit/rdm/migration/transform/transform.py

```diff
@@ -304,21 +311,54 @@ def creator_identifiers(creator):
             inner_dict = creator.get("person_or_org", {})
             identifiers = inner_dict.get("identifiers", [])
             for identifier in identifiers:
-                # TODO process CDS and CERN Ids when names vocabulary ready
-                if identifier["scheme"] == "inspire":
+                # we check for unknown schemes
+                if identifier["scheme"] in ["inspire", "orcid", "cern", "cds"] :
```

Comment:
Can we compute this from one of our configs? Maybe the `RDM_RECORDS_PERSONORG_SCHEMES`?

Reply:
I would prefer not to rely on the config, in order to control better what goes through in migration.

Reply:
But I would rely on the config for the exact same reason :) Let's discuss it IRL...